src/modules/projectImageContext.js

```diff
@@ -0,0 +1,31 @@
+class ProjectImageContext {
+
+    constructor(sessionManager, projectId, imageId) {
+        this.sessionManager = sessionManager;
+        this.projectId = projectId;
+        this.imageId = imageId;
+    }
+
+    /**
+     * Get the project image.
+     *
+     * @returns The project image
+     */
+    get = () => this.sessionManager.gatekeeper(`/projects/${this.projectId}/images/${this.imageId}`).then((response) => response.json());
```

Comment:
Maybe to align the sessionManager weith the naming scheme, call it `HanskenContext`?

`this.hanskenContext.gatekeeper(...);`

Reply:
Discussed. The user doesn't see this object, and the only goal of the object is to keep a valid session with any of the Hansken services.